build.sh

```diff
@@ -0,0 +1,11 @@
+cp ./p0.js ./build/p0.js
+
+# See https://nodejs.org/docs/latest-v20.x/api/single-executable-applications.html for more information.
+# Some of these commands only work on macOS. Windows and Linux require slightly different commands.
```

Comment:
Does this mean that the executable we build is only runnable on mac machines with our same chip: `Apple M3`? 🤔

Reply:
The way it works is instead of building an executable from scratch, it uses whatever version of NodeJS is already installed on your system and putting your custom application code (in this case our CLI) into a copy of the existing node binary. So my guess is that the built P0 CLI would be runnable on machines that the underlying Node binary is compatible with.

Reply:
I wonder if this means we'll need to start trying to build and host deployable's somewhere 🤔.

I don't think we need to worry about distribution at this time but something to keep in mind going forward. Like if we publish an update we'll need CI/CD to create a deployable, save it somewhere, and we'll need to make sure the version check works with the packaged executable and tells the user correctly where to download the latest version from.